nuxeo-mongodb-directory/src/main/java/org/nuxeo/directory/mongodb/MongoDBSession.java

```diff
@@ -167,10 +255,32 @@ public void deleteEntry(DocumentModel docModel) throws DirectoryException {
 
     @Override
     public void deleteEntry(String id) throws DirectoryException {
+        deleteEntry(directoryName, id);
+    }
+
+    @Override
+    public void deleteEntry(String id, Map<String, String> map) throws DirectoryException {
```

Comment:
The `map` is ignored here. There should at least be a big

    // TODO implement delete with secondary ids

Or if you think this method is useless because unused in the platform or for another reason,

    // TODO deprecate this, it's, unused